waku/v2/node/wakunode2.go

```diff
@@ -241,7 +243,12 @@ func (w *WakuNode) checkForAddressChanges() {
 }
 
 func (w *WakuNode) Start() error {
-	w.store = store.NewWakuStore(w.host, w.opts.messageProvider, w.opts.maxMessages, w.opts.maxDuration)
+	w.swap = swap.NewWakuSwap([]swap.SwapOption{
```

Comment:
Just a question:
Would it be possible to create a store node without enabling Swap?

Reply:
If you build the store node manually, you can pass nil. If you use wakunode2 I didn't allow this option as by default, this is soft mode which does only accounting (inc/dec a counter). I don't think this would be problematic, let me know if you want this option or if you think this is problematic